spec/features/06_user_views_all_shows_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+feature "visitors can view shows on index page" do
+  before(:each) do
+    @shows_list = []
+    2.times do
+      @shows_list << FactoryGirl.create(:show)
+    end
+  end
+
```

Comment:
Your tests are going to be FAR more telling if you create the two shows individually, and set them to a variable. Then simply call that variable in your expect statements, rather than relying on a show list array